evcouplings/compare/protocol.py

```diff
@@ -770,6 +767,9 @@ def _identify_monomer_structures(name_prefix, outcfg, aux_prefix):
     first_segment_name = kwargs["segments"][0][0]
     second_segment_name = kwargs["segments"][1][0]
```

Comment:
Can you please also add `segments` to the list of parameters checked using check_required above and check why we have `first_segments` and `second_segments` which are not used? Or maybe first_segments and second_segments are what we would like to use here?

Reply:
Oops yeah it doesn't matter which one we use but they need to be consistent